.github/CODEOWNERS

```diff
@@ -153,7 +153,7 @@ peps/pep-0293.rst  @doerwalter
 # peps/pep-0295.rst
 # peps/pep-0296.rst
 peps/pep-0297.rst  @malemburg
-peps/pep-0298.rst  @theller
+peps/pep-0298.rst
```

Comment:
Should it be commented out instead like the next entry?

Reply:
Seems like it's valid without commenting out, but yes, good idea, let's keep them consistent.

```suggestion
# peps/pep-0298.rst
```